procrastinate/jobs.py

```diff
@@ -136,7 +137,8 @@ def evolve(self, **kwargs: Any) -> Job:
     @cached_property
     def call_string(self):
         kwargs_string = ", ".join(
-            f"{key}={value!r}" for key, value in self.task_kwargs.items()
+            f"{key}={ellipsize_middle(repr(value))}"
+            for key, value in self.task_kwargs.items()
```

Comment:
I think you left out the part where we allow people to override this. It might be ok in this PR, but I'd like to make sure we're fine with this before a release. Doesn't have to be something gigantic, but this may be ruining someone's workflow with no hopes of improvement.

Reply:
Ah I just saw your comment in the PR body ! I'll try to think about it. It's ok to leave it out of the PR for now if it's complicated.

Reply:
Maybe it's easier if it can be configured in the tasks, then ?

Reply:
This actually made me thing... Perhaps a better solution to all of this would be to have a way to simply exclude an argument from display (`call_string`, log context, and Django admin), for whatever reason (security, size).

This seems much simpler than all of this.

What do you think?

Reply:
Ugh, it seemed much simpler but:
- We can define on `Task.unlogged_args`, but you can't get from `Job` -> `Task` easily
- Seems wrong to persist fixed argument names into `Job`/database
- Also seems fairly "dangerous": you forget to filter in one place (call string, log context, admin, shell) and you're printing what you should not be.
- Perhaps splitting `task_kwargs` into `task_kwargs`+`task_unlogged_kwargs` but that seems like an overkill too.